cmd/vsphere-xcopy-volume-populator/internal/populator/secure-vmkfstools-wrapper.py

```
@@ -0,0 +1,311 @@
+        print(f">>>>>>>>> Executing operation: {operation} with args: {arg1}, {arg2}", file=sys.stderr)
+        
+        if operation == "clone":
+            clone_operation(arg1, arg2)
+        elif operation == "status":
+            status_operation(arg1)
+        elif operation == "cleanup":
+            cleanup_operation(arg1)
+        else:
+            raise ValueError(f"Invalid operation: {operation}")
+            
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)